amida-tech/amida-auth-microservice
# Feature/jwt payload

#### What's this PR do?
Add email and scopes to the JWT payload.

#### How should this be manually tested?
Login with a user, decode the returned JWT, and verify that it contains username, email, and auth scope information for the user.
